evalai/utils/submissions.py

```diff
@@ -71,3 +109,54 @@ def display_my_submission_details(challenge_id, phase_id):
 
     submissions = response["results"]
     pretty_print_my_submissions_data(submissions)
+
+
+def pretty_print_submission_details(submission):
+    """
+    Function to print details of a submission
+    """
+    team_name = "\n{}".format(style(submission['participant_team_name'], bold=True, fg="green"))
+    sid = "Submission ID: {}\n".format(style(str(submission['id']), bold=True, fg="blue"))
+    team_name = "{} {}".format(team_name, sid)
+
+    status = style("\nSubmission Status : {}\n".format(submission['status']), bold=True)
+    execution_time = style("\nExecution Time (sec) : {}\n".format(submission['execution_time']), bold=True)
+
+    # Converting UTC to local timezone
+    date = datetime.strptime(submission['submitted_at'], "%Y-%m-%dT%H:%M:%S.%fZ")
+    from_zone = tz.tzutc()
+    to_zone = tz.tzlocal()
+
+    date = date.replace(tzinfo=from_zone)
+    converted_date = date.astimezone(to_zone)
```

Comment:
possibly put this in a separate `utils` module? I expect one to use it for a lot of similar things.